src/main/java/net/dv8tion/jda/core/managers/impl/AudioManagerImpl.java

```diff
@@ -65,8 +66,8 @@
 
     public AudioManagerImpl(Guild guild)
     {
-        this.guild = guild;
-        this.api = (JDAImpl) guild.getJDA();
+        this.guild = (GuildImpl) guild;
```

Comment:
Guild should only ever be a GuildImpl; its an interface to prevent users from making their own Guild objects